### DISCUSSION TITLE
crazyradio2 use Nordic PCA10056 board（nRF52840 chip)，but cfclient disconnect crazyflie2.1, Use crazyradio PA connect normally

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1623] [General]
Hi,
I download the crazyradio2 bootloader and firmware to Nordic PCA10056 board（nRF52840 chip）, the windows device manager display the **Crazyradio PA USB dongle**，but cfclient disconnect crazyflie2.1, Use crazyradio PA connect normally, Why ?

![image](https://github.com/user-attachments/assets/a2446593-e81a-49ea-9b1a-7047503ed963)
![image](https://github.com/user-attachments/assets/be703a14-5962-417c-a085-8477fed66ecc)

**Comment by ToveRumar** [comment]
Hi!

Thank you for your question!

It seems you are using custom hardware, is this correct?
We can unfortunately not provide any support for custom hardware. 

I wish you good luck with your project!

**Reply by justinleeyang** [reply]
It isn't custom hardware, it is nordic's development board. because no have nRF21540 chip ?

**Reply by ataffanel** [reply]
It is custom in the sense that it is not the Crazyradio 2.0 hardware but a devboard, we can only provide very limited support since we do not support this devboard officially.

That said, I can try to help you figure out what is going on.

Did you map the LEDs on the dev-board LEDs and are these LED behaving correctly? When scanning for a Crazyflie, one of the LED should be lit most of the time (the one that is RED on the Crazyradio).
